### PR TITLE
ci: add issue templates

### PR DESCRIPTION
**Pull request by matgorzynski**

### CODE REVIEW

File: ISSUE_TEMPLATE/other.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: Other
+about: Other type of issue
+title: '[Other]'
+labels: other
+assignees: ''
+
+---
+
+**Please provide informaton**
+Please describe, in as much detail as you can, the situation with which you are addressing us
```
Comment (adr7dev) [inline ISSUE_TEMPLATE/other.md:11]: Dot at the end of the line.

File: ISSUE_TEMPLATE/request-support.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+labels: support
+assignees: ''
+
+---
+
+**Describe the problem, which you need a support**
+Describe your problem and what exactly you need help with.
+
+**Which repository do you need help with**
+E.g, ct-web
```
Comment (adr7dev) [inline ISSUE_TEMPLATE/request-support.md:14]: Replace with "E.g."

File: ISSUE_TEMPLATE/other.md (hunk 1 above)
Comment (matgorzynski) [reply ISSUE_TEMPLATE/other.md]: added

File: ISSUE_TEMPLATE/request-support.md (hunk 2 above)
Comment (matgorzynski) [reply ISSUE_TEMPLATE/request-support.md]: replaced

**Review by adr7dev — APPROVED** [review]

**Pull request merged**